packages/bpk-mixins/src/mixins/_layers.scss

```diff
@@ -74,19 +74,24 @@
 ///     @include bpk-layer-arrow-top(1rem);
 ///   }
 
-@mixin bpk-layer-arrow-top($size) {
+@mixin bpk-layer-arrow-top($arrow-size) {
   bottom: 100%;
-  left: 50%;
+  width: ($arrow-size * 2);
+  height: $arrow-size;
 
   &::before {
-    top: -($size + $bpk-one-pixel-rem);
-    margin-left: -($size + $bpk-one-pixel-rem);
+    top: -($arrow-size + $bpk-one-pixel-rem);
+    margin-left: -$bpk-one-pixel-rem;
     border-bottom-color: $bpk-color-gray-100;
+
+    @include bpk-rtl {
+      margin-right: -$bpk-one-pixel-rem;
+      margin-left: 0;
```

Comment:
This was always broken,

Reply:
I think I need more context here - it was always broken and now it's not, or it's still broken?

Reply:
Of course - so the arrow border in RTL was always borked - fixed it.
![image](https://user-images.githubusercontent.com/1440090/34766871-f7fa6882-f5ed-11e7-81de-ea5da402e19f.png)